Float.Core/Net/RequestClient.cs

```diff
@@ -280,9 +298,17 @@ async Task<Response> Send(HttpRequestMessage request)
 
             try
             {
+                var originalUri = request.RequestUri?.AbsolutePath;
+
                 var httpResponse = await webClient.SendAsync(request).ConfigureAwait(false);
                 var responseContent = await httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
 
+                // Verify if a forbidden redirect happened.
+                if (httpResponse.StatusCode == HttpStatusCode.Forbidden && originalUri != request.RequestUri?.AbsolutePath)
+                {
+                    throw new ForbiddenRedirectException(request.RequestUri);
```

Comment:
Instead of assuming the server is going to return a 403 forbidden, can we avoid following the redirect by default (so we can get the 301 or 302 status code) and then manually follow the redirect?

Reply:
unfortunately seems that 301/302 only happens with a successful redirect in this case the redirect is failing and we're getting a different status code now Im just verifying if the status code is not success